mai/go.mod

```diff
@@ -0,0 +1,80 @@
+module github.com/nakji-network/connectors/mai
+
+go 1.20
+
+replace github.com/nakji-network/connector => ../../connector
```

Comment:
this could be the reason why the build is failing

Reply:
this one is forgotten there but there were other reasons too 🙂 thanks